internal/pkg/plan/persist_test.go

```diff
@@ -137,7 +137,7 @@ func TestPersistNewConfig(t *testing.T) {
 						ParentPath: "main",
 						ObjectPath: "extractor/ex-generic-v2/new-config",
 					},
-					RelatedPaths: []string{model.MetaFile, model.DescriptionFile, model.ConfigFile},
+					RelatedPaths: []string{model.MetaFile, model.ConfigFile, model.DescriptionFile},
```

Comment:
Commit 24: 35d3a030074c5b5606d4e68e3aefa49dda7db15a

Subory sa nacitavaju v inom poradi, ... nema to na nic vplyv, ale musi sa to v testoch opravit.